src/wp-content/themes/twentytwentytwo/style.css

```
@@ -158,4 +158,4 @@ body > .is-root-container > .wp-block-template-part > .wp-block-cover,
 
 .wp-block-post-comments ol.commentlist ul.children li:not(:last-child) {
 	margin-bottom: 1rem;
-}
+}
```

```suggestion
}

```

Add missing end line.